app/controllers/collection/index.js

```diff
@@ -41,26 +41,21 @@ exports.uploadCollectionImage = multer({
 
 exports.addCollection = async (req, res) => {
```

Comment:
We don't need to CRUD collections in backend. They are already in smart contract. If it is needed, you need to listen.
Frontend doesn't make API for collection updates.
You can create collection thru etherscan too. Right?